django/contrib/gis/db/backends/postgis/operations.py

```diff
@@ -140,18 +140,13 @@ def select(self):
 
     select_extent = None
 
-    def __init__(self, connection):
-        super().__init__(connection)
-
-        prefix = self.geom_func_prefix
-
-        self.collect = prefix + 'Collect'
-        self.extent = prefix + 'Extent'
-        self.extent3d = prefix + '3DExtent'
-        self.length3d = prefix + '3DLength'
-        self.makeline = prefix + 'MakeLine'
-        self.perimeter3d = prefix + '3DPerimeter'
-        self.unionagg = prefix + 'Union'
+    collect = geom_func_prefix + 'Collect'
```

Comment:
These might be sorted somewhere above `gis_operators` for similarity with the ordering in `spatialite/operations.py`.